### PR TITLE
Added basic layout with sidebar and routing

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Files to check:
App.js
App.css
Header.jsx
Header.css
Sidebar.jsx
Sidebar.css

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+import { Routes, Route } from "react-router-dom";
+import { Header, Sidebar } from "./components";
+import {
+	Home,
+	VideoListing,
+	PlayList,
+	WatchLater,
+	History,
+	LikedVideos,
+} from "./pages";
```
Comment (Naman-Saxena1) [inline src/App.js:11]: Can export all from index.js and then import here from 1 file

File: src/icons.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	FaBell,
+	FaUserCircle,
+	FaHome,
+	FaCompass,
+	FaThList,
+	FaThumbsUp,
+	FaThumbsDown,
+	FaHistory,
+	FaClock,
+	FaYoutube,
```
Comment (Naman-Saxena1) [inline src/icons.js:32]: I liked how you are exporting all icons from 1 file

File: src/App.js [hunk 3]
```diff
@@ -1,7 +1,35 @@
...
 function App() {
-	return <div className="App">Hi</div>;
+	return (
+		<>
+			<Header />
+			<div className="content-section">
+				<aside className="aside">
+					<Sidebar />
+				</aside>
```
Comment (Naman-Saxena1) [inline src/App.js:19]: Nice, you used aside tag for it 👍

**Review by Naman-Saxena1 — APPROVED** [review]
Can't find any major issues, looks good to merge

File: src/components/header/Header.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	padding: 0 1rem;
+}
+.btn-icon {
+	color: #000;
+}
+@media (max-width: 768px) {
+	.hide {
+		display: none;
+	}
+}
```
Comment (Sahil25061999) [inline src/components/header/Header.css]: use em

File: src/components/sidebar/Sidebar.jsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+				<FaClock size={20} /> <p>Watch Later</p>
+			</Link>
+			<Link
+				to="/history"
+				className={`sidebar-item link-no-style + ${
+					location === "/history" ? "active" : ""
+				}`}
+			>
+				<FaHistory size={20} /> <p>History</p>
+			</Link>
```
Comment (Sahil25061999) [inline src/components/sidebar/Sidebar.jsx]: you can use NavLink for the rest too

**Review by Sahil25061999 — APPROVED** [review]
didnt find major issues, mostly looks good ,  loved the idea of creating components for the icons.

File: src/App.js (hunk 1 above)
Comment (nikhiljugale007) [reply src/App.js:11]: Yes. I am exporting from common index.js. like from "./pages/index.js'

File: src/components/header/Header.css (hunk 4 above)
Comment (nikhiljugale007) [reply src/components/header/Header.css]: used em .

File: src/components/sidebar/Sidebar.jsx (hunk 5 above)
Comment (nikhiljugale007) [reply src/components/sidebar/Sidebar.jsx]: Used NavLink for rest to.

**Pull request merged**